client/src/views/sidebar-menu/SidebarMenuPage.vue

```diff
@@ -791,6 +794,7 @@ ion-menu {
       display: flex;
       align-items: center;
       padding: 0.125rem 0;
+      cursor: pointer;
```

Comment:
The Favorites section also uses this css class, either differentiate the two or add the `@click` on the favorites
![Screenshot 2024-10-16 at 16 04 43](https://github.com/user-attachments/assets/e776987b-a2d1-47f6-9177-a200e42b038c)

Reply:
separate the css with two class, put pointer only on "Watcher's Keep" not the Favorites title